### ISSUE TITLE
Remove custom seo.html

### ISSUE THREAD

**Issue by SeanKilleen** [SeanKilleen/seankilleen.github.io#310]
Once my PR is merged into the minimal-mistakes repo or they move to jekyll-seo-tag.

Adding this here so I don't forget that I forked the theme a little bit.

**Comment by SeanKilleen** [comment]
My changes are available as of `v4.15.0` so I should be able to update & delete the file. Confirmed on 4/2/2018 that the contents of the files match except for my comments.

**Comment by SeanKilleen** [comment]
This has been done.
